docs/authorization.md

```diff
@@ -230,6 +230,41 @@ i.e. a claimed resource is bound to the same maximal permission policy. Only the
 
 TBD: Example
 
+## Consumer Permission Claims Authorizer
+
+The requested resource type might be a claimed resource of a PermissionClaim in an APIExport
+referenced by an APIBinding in the workspace being accessed.
+In this case access to the requested resource is constrained by the field `consumerVerbs` of the underlying permission claim.
+
+If the requested resource type is claimed by different API Exports in the same workspace,
+the intersection of all verbs of all APIExports is being granted to the user.
+
+Valid values for the `consumerVerbs` field follow the same rules as the `verbs` field in PolicyRules:
+
+- It is required.
+- `"*"` matches all verbs. If present, it must be the only entry.
+- The list must not be empty.
+
+If the APIExport of the requested and claimed resource has a maximum permission policy configured,
+access to the requested resource can be further constrained as explained in the "API binding authorizer" section.
+
+Example: Given an APIExport `foo` consumer are supposed to be given read-only access to the claimed resource `bar.some.group`:
+
+```yaml
+apiVersion: apis.kcp.dev/v1alpha1
+kind: APIExport
+metadata:
+  name: foo
+spec:
+...
+  permissionClaims:
+  - resource: bars
+    group: some.group
+    consumerVerbs: ["get", "watch", "list"]
+    providerVerbs: ["*"]
```

Comment:
provider == claimer actually. Usually claimer != owner of the export. The wording need more clarity I think.

Also, can a claimer remove permission of the API owner?